index.js

```diff
@@ -0,0 +1,98 @@
+//DOM ELEMENTS
+const playerImg = document.querySelector(".player img")
+const h2Name = document.querySelector(".player h2")
+const emNickname = document.querySelector(".player em")
+const pLikes = document.querySelector(".player p")
+const goalsUl = document.querySelector("#goals")
+const likeButton = document.querySelector(".like-button")
+const goalForm = document.querySelector("#new-goal-form")
+
+
+//FETCH DATA
+function getData() {
+    return fetch("http://localhost:3000/players/1")
+        .then(resp => resp.json())
+        .then(playerObj => renderPlayer(playerObj))
+}
+getData()
+
+//Event Listeners
+
+likeButton.addEventListener("click", getLikes)
+goalForm.addEventListener("submit", newGoal)
+
+//Event Callbacks
+
+function newGoal(event) {
+    event.preventDefault()
+    const newGoalObj = {
+        // id: 5,
+        playerId: parseInt(event.target.previousElementSibling.dataset.id),
+        link: event.target['goal-link'].value,
+        description: event.target['goal-description'].value
+    }
+    // debugger
+    fetch("http://localhost:3000/goals", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(newGoalObj)
+    })
+    .then(resp => resp.json())
+    .then(updatedGoals => {
+        console.log('success', updatedGoals)
+    })
+    createGoal(newGoalObj)
```

Comment:
Since your `createGoal` function expects the goal object to have an ID, this won't quite work - this is a case where you should use pessimistic rendering instead, since the server will send back an object with an ID:

```js
    .then(updatedGoals => {
        createGoal(updatedGoals)
    })
``